lib/generators/react_on_rails/generator_helper.rb

```diff
@@ -48,3 +56,19 @@ def copy_file_and_missing_parent_directories(source_file, destination_file = nil
     copy_file source_file, destination_file
   end
 end
+
+module GeneratorErrors
```

Comment:
@aaronvb Let's put this in it's own file. Any reason to put in generator_helper.rb?